### PR TITLE
[CND] small-1.3 Remaining licences in cnd cluster

### PR DESCRIPTION
**Pull request by vieiro**
Resolves some more license issues that were not detected by rat.

### CODE REVIEW

File: cnd/cnd/javahelp/org/netbeans/modules/cnd/help/CandC++ProjectBasics/AddtlIncludeDirectories.html [hunk 1]
```diff
@@ -46,7 +46,7 @@ <h2>
     </ol>
     <hr>
     <p>
-       <small><a href="nbdocs:/org/netbeans/modules/usersguide/legal_notice.htm">Copyright&nbsp;&copy;&nbsp;2015,&nbsp;Oracle&nbsp;and/or&nbsp;its&nbsp;affiliates.&nbsp;All&nbsp;rights&nbsp;reserved.</a></small>
+       <small>
```
Comment (matthiasblaesing) [inline cnd/cnd/javahelp/org/netbeans/modules/cnd/help/CandC++ProjectBasics/AddtlIncludeDirectories.html:49]: The HTML is now broken, `<small>` is opened, but not closed. I would remove the `<small>` and `<p>` element. This is present in multiple HTML files.

**Comment by vieiro** [comment]
I think we're done with cmd-small/1 in https://cwiki.apache.org/confluence/display/NETBEANS/cnd+integration+proposal%3A+cnd-small.

**Review by matthiasblaesing — COMMENTED** [review]
Sorry, there are more issues in the HTML files (caused by this modification):

Opening small without closing:
cnd/cnd/javahelp/org/netbeans/modules/cnd/help/WorkingWithCandC++Projects/SettingMainCorC++Project.html

Empty link (no text):
cnd/cnd/javahelp/org/netbeans/modules/cnd/help/WorkingWithCandC++Projects/WorkingWithJNI.html

I suggest not try to remove the copyright text, but replace it with "Legal notice". That way there will be no broken elements (the HTML structure does not need to be touched) and there will be no empty links.

**Comment by vieiro** [comment]
Yes, let's do that!

**Comment by vieiro** [comment]
Will make a new PR.

**Pull request closed without merging**